packages/trader/src/AppV2/Components/Filter/time-filter.tsx

```diff
@@ -71,34 +76,46 @@ const TimeFilter = ({
         setIsDropdownOpen(false);
 
         if (value === 'Today') {
```

Comment:
you cna turn this into an array of objects with value being key and can have default case when values dont match key in the object

Reply:
You mean something  like this?
<img width="1150" alt="Screenshot 2024-06-07 at 9 27 11 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/e4cc4deb-bef8-4912-b8cd-7ad614fa3453">

Reply:
Applied, thank you)